apps/insights/src/hooks/use-query-param-filter-pagination.ts

```diff
@@ -14,8 +14,9 @@ import { useCallback, useMemo } from "react";
 
 export const useQueryParamFilterPagination = <T>(
   items: T[],
-  predicate: (item: T, term: string) => boolean,
+  predicate: (item: T, search: string) => boolean,
   doSort: (a: T, b: T, descriptor: SortDescriptor) => number,
+  doMutate: (items: T[], search: string) => T[],
```

Comment:
Could probably have deleted `predicate` and `doSort` at this point since `doMutate` is a more open/powerful API anyway